tests/php/sync/test_class.jetpack-sync-comments.php

```diff
@@ -76,6 +76,25 @@ public function test_modify_comment_content() {
 		$this->modify_comment_helper( $comment, $expected_variable );
 	}
 
+	public function test_do_not_sync_comment_with_unknow_comment_type() {
+		$this->server_event_storage->reset();
+		$comment_data = array(
+			'comment_post_ID' => $this->post_id,
+			'comment_date' => date('Y-m-d H:i:s', time() ),
+			'comment_date_gmt' => date('Y-m-d H:i:s', time() ),
+			'comment_author' => 'ActionScheduler',
+			'comment_content' => 'fun!',
+			'comment_agent' => 'ActionScheduler',
+			'comment_type' => 'action_log',
+		);
+		wp_insert_comment( $comment_data );
+		$this->sender->do_sync();
+
+		$event = $this->server_event_storage->get_most_recent_event( 'wp_insert_comment' );
+		$this->assertFalse( $event );
```

Comment:
Can we add a test case to ensure that a comment with a whitelisted comment type will actually be synced?

Reply:
Done.